runtime/jaktlib/platform.jakt

```diff
@@ -38,8 +36,16 @@ function platform_import_names() throws -> [String] {
     }
 }
 
+function add_to_each(anon strings: [String], prefix: String, suffix: String) throws -> [String] {
+    mut output: [String] = []
+    for str in strings {
+        output.push(prefix + str + suffix)
+    }
+    return output
+}
+
 comptime platform_module(anon name: String) throws -> [String] {
-    return append_to_each(platform_import_names(), suffix: format("_{}", name))
+    return add_to_each(platform_import_names(), prefix: "jakt::platform::", suffix: format("_{}", name))
```

Comment:
This seems a bit odd to me, you'd expect
```
import platform_module("foo")
```
to read "import platform-dependent module foo" and not "import platform-dependent _standard_ module foo"
perhaps this should accept something like
```
import platform_module("jakt::platform::foo")
```
and transform it into `jakt::platform::something_foo`, which would allow the user to also say `import platform_module("foo")` and get `posix_foo`.

Reply:
Yeah that makes sense, have updated it to work like that.